apps/src/templates/studioHomepages/CoteacherInviteNotification.jsx

```diff
@@ -36,55 +43,59 @@ const CoteacherInviteNotification = ({
       .catch(err => console.error(err));
   };
 
-  const acceptCoteacherInvite = id => {
+  const acceptCoteacherInvite = (id, sectionId) => {
     analyticsReporter.sendEvent(EVENTS.COTEACHER_INVITE_ACCEPTED, {
-      sectionId: coteacherInvite.section_id,
+      sectionId: sectionId,
     });
     buttonAction(`/api/v1/section_instructors/${id}/accept`);
   };
 
-  const declineCoteacherInvite = id => {
+  const declineCoteacherInvite = (id, sectionId) => {
     analyticsReporter.sendEvent(EVENTS.COTEACHER_INVITE_DECLINED, {
-      sectionId: coteacherInvite.section_id,
+      sectionId: sectionId,
     });
     buttonAction(`/api/v1/section_instructors/${id}/decline`);
   };
 
-  return (
-    <Notification
-      dismissible={false}
-      type={NotificationType.collaborate}
-      iconStyles={styles.icon}
-      notice={i18n.coteacherInvite({
-        invitedByName: coteacherInvite.invited_by_name,
-      })}
-      details={
-        <BodyTwoText style={{marginBottom: 0}}>
-          {i18n.coteacherInviteDescription({
-            invitedByEmail: coteacherInvite.invited_by_email,
-          })}
-          <br />
-          <StrongText>{coteacherInvite.section_name}</StrongText>
-        </BodyTwoText>
-      }
-      tooltipText={i18n.coteacherTooltip()}
-      buttonsStyles={styles.buttons}
-      buttons={[
-        {
-          text: 'Decline',
-          onClick: () => declineCoteacherInvite(coteacherInvite.id),
-          color: Button.ButtonColor.neutralDark,
-          style: styles.declineButton,
-        },
-        {
-          text: 'Accept',
-          onClick: () => acceptCoteacherInvite(coteacherInvite.id),
-          color: Button.ButtonColor.brandSecondaryDefault,
-          style: styles.acceptButton,
-        },
-      ]}
-    />
-  );
+  if (invite) {
```

Comment:
nit: still a slight preference for `if(!invite) {return null}` so that it is clearer what has changed (no whitespace changes) and also to decrease amount of tabs in general, but up to you.